pkg/tenant/tenant.go

```diff
@@ -352,6 +352,7 @@ func (s *Server) Routes(router *gin.Engine) (err error) {
 			topics.GET("", mw.Authorize(perms.ReadTopics), s.TopicList)
 			topics.POST("", csrf, mw.Authorize(perms.EditTopics), s.TopicCreate)
 			topics.GET("/:topicID", mw.Authorize(perms.ReadTopics), s.TopicDetail)
+			topics.GET("/:topicID/events", mw.Authorize(perms.ReadTopics, perms.ReadMetrics), s.TopicEvents)
```

Comment:
Not sure if there's a better name of this, but it's what the frontend is using right now.